var/spack/repos/builtin/packages/py-python-libsbml/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPythonLibsbml(PythonPackage):
+    """LibSBML is a library for reading, writing and manipulating the Systems Biology
+    Markup Language (SBML)."""
+
+    homepage = "https://sbml.org/"
+    git = "https://github.com/sbmlteam/python-libsbml.git"
+    pypi = "python-libsbml/python-libsbml-5.19.7.tar.gz"
+
+    version("5.19.7", sha256="447b1fde7aceccd11a93dc9f589ffd9319ba854d7b7583f911259a8b0127ab7b")
+    version("5.19.5", tag="v5.19.5", submodules=True)
+
+    depends_on("py-setuptools", type="build")
+
+    depends_on("swig", type=("build", "run"))
+    depends_on("cmake", type=("build", "run"))
```

Comment:
These are rarely needed at run-time. Where did you find them listed?

Reply:
https://github.com/sbmlteam/python-libsbml#preparing-the-package-build
I'm never sure whether something like this means it should or shouldn't be there at runtime, but I can remove the "run" type.

Reply:
I think those are only needed to build, you can try using it and see if anything breaks if these aren't available in PATH if you want to confirm.

Reply:
AMICI itself needs `cmake` at runtime, as it generates a C++ model from SBML. Then it runs `setup.py` and BLAS/SWIG/HDF5/CMake to build a Python module from the C++ code.